src/main/webapp/app/overview/course-lectures/course-lecture-details.component.html

```diff
@@ -19,6 +19,13 @@ <h4>
                     </span>
                 </h4>
             </div>
+            <ng-container *ngIf="showGoToLectureManagementButton()">
+                <div class="col-auto wrapper-btn-manage-lecture">
+                    <button id="manageLectureButton" class="btn btn-manage-lecture" (click)="redirectToLectureManagement()">
```

Comment:
Is there any reason an `id` is needed here? I think the CSS class `btn-manage-lecture` would suffice to find the button in the client tests.

Reply:
Now with common class used I makes sense to have id here

Reply:
I don't quite understand 🤔 Which class is common? Are there two elements with the class `btn-manage-lecture` in the component?  Maybe you can just use a more specific CSS selector instead?

Reply:
It now uses `btn-secondary` class instead of `btn-manage-lecture`. We cannot assume that there won't be other button using it in this component

Reply:
I'm not really a fan of HTML ids in general, but if it's really necessary here I think it's ok.